### ISSUE TITLE
Running error

### ISSUE THREAD

**Issue by KasitC** [mskcc/vcf2maf#335]
Hello,

I tried running vcf2maf but got this error, would you mind helping me with this?

Many thanks in advance!

Best wishes,
Kasit

```
perl vcf2maf/vcf2maf.pl --input-vcf /Users/kasit/Desktop/HN12-T-sample.b37-filtered.vcf --output-maf /Users/kasit/Desktop/HN12-T-sample.b37-filtered.maf --vep-path /Users/kasit/Desktop/KC/Bioinformatic_programs/ensembl-vep/ --vep-data /srv/vep
STATUS: Running VEP and writing to: /Users/kasit/Desktop/HN12-T-sample.b37-filtered.vep.vcf
Unknown option: af_esp
ERROR: Failed to parse command-line flags

ERROR: Failed to run the VEP annotator! Command: /Users/kasit/perl5/perlbrew/perls/perl-5.26.2/bin/perl /Users/kasit/Desktop/KC/Bioinformatic_programs/ensembl-vep//vep --species homo_sapiens --assembly GRCh37 --no_progress --no_stats --buffer_size 5000 --sift b --ccds --uniprot --hgvs --symbol --numbers --domains --gene_phenotype --canonical --protein --biotype --uniprot --tsl --variant_class --shift_hgvs 1 --check_existing --total_length --allele_number --no_escape --xref_refseq --failed 1 --vcf --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --dir /srv/vep --fasta /Users/kasit/.vep/homo_sapiens/102_GRCh37/Homo_sapiens.GRCh37.dna.toplevel.fa.gz --format vcf --input_file /Users/kasit/Desktop/HN12-T-sample.b37-filtered.vcf --output_file /Users/kasit/Desktop/HN12-T-sample.b37-filtered.vep.vcf --offline --pubmed --fork 4 --polyphen b --af --af_1kg --af_esp --af_gnomad --regulatory
```

**Comment by guodudou** [comment]
@KasitC I got exactly the same error. Have you had this problem fixed?

Thanks,
Wendy

**Comment by andrew2k** [comment]
Same error.

**Comment by chenzemin** [comment]
I change vcf2maf.pl source code as follows：

before：
`$vep_cmd .= ( $vep_script =~ m/vep$/ ? " --af_1kg --af_esp --af_gnomad" : " --maf_1kg --maf_esp" ) unless( $online );`

after:
`$vep_cmd .= ( $vep_script =~ m/vep$/ ? " --af_1kg --force_overwrite --af_gnomad" : " --maf_1kg --maf_esp" ) unless( $online );`

and that work for me.

**Comment by famosab** [comment]
Are there any updates on plans to fix this? :)

**Comment by ckandoth** [comment]
This was fixed in [v1.6.22](https://github.com/mskcc/vcf2maf/releases/tag/v1.6.22).
